src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_DebugInfoScreen.tsx

```diff
@@ -619,27 +617,6 @@ export const Profile_DebugInfoScreen = () => {
                     style={style.button}
                     text="Onboard"
                   />
-                  <Button
-                    interactiveColor="interactive_0"
-                    onPress={async () => {
-                      await startBeacons();
-                    }}
-                    style={style.button}
-                    disabled={
-                      kettleInfo?.isKettleStarted ||
-                      !kettleInfo?.isBeaconsOnboarded
-                    }
-                    text="Start"
-                  />
-                  <Button
-                    interactiveColor="interactive_0"
-                    onPress={async () => {
-                      await stopBeacons();
```

Comment:
Should `revokeBeacons` be here instead of completely removing it?

Reply:
There is a button in the debug that revokes.